apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/add-authenticator-modal.tsx

```diff
@@ -535,22 +536,22 @@ export const AddAuthenticatorModal: FunctionComponent<AddAuthenticatorModalProps
                                             templateIndex: number
                                         ) => {
 
-                                            const isOrgIdp: boolean =
-                                                template.templateId === "organization-enterprise-idp";
-
-                                            if (isOrgIdp && !isOrganizationManagementEnabled) {
-                                                return null;
-                                            }
-
-                                            if (isOrgIdp && orgType === OrganizationType.SUBORGANIZATION) {
+                                            if (template.templateId ===
+                                                    ConnectionManagementConstants.IDP_TEMPLATE_IDS.LINKEDIN
+                                                || template.templateId ===
+                                                    ConnectionManagementConstants.IDP_TEMPLATE_IDS
+                                                        .ORGANIZATION_ENTERPRISE_IDP
+                                                || template.templateId ===
+                                                    ConnectionManagementConstants.TRUSTED_TOKEN_TEMPLATE_ID
+                                            ) {
```

Comment:
shall we include these IDs in an array and use `.includes`? 

It will improve readability i.e. the reader will know why null is returned for these IDs, and it'd be scalable.

eg: 

const templates = [linkedin_idp_id, org_enperprise_id,tti_id]
if (templates.includes(template?.templateId)) {
   return null
}